### DISCUSSION TITLE
Turtle Mode on Inav 5.1

### DISCUSSION THREAD

**Discussion by koklimabc** [iNavFlight/inav#8850] [Q&A]
So far I'm using SpeedyBee F405V3 under Inav 5.1 but I din't able to find the turtle mode option under Mode Tab.

Is there anywhere to set it up?

**Comment by Jetrell** [comment]
It only works if the ESC has Dshot capability..
Then you need to have `ESC output/telemetry` selected under Peripherals in the Ports tab. 
And a Dshot protocol has to be selected under the Outputs tab..  
Only then will Turtle appear under the modes tab.

**Reply by breadoven** [reply]
Shouldn't need for ESC telemetry surely. Works OK with just Dshot selected in my case.

**Reply by Jetrell** [reply]
> Shouldn't need for ESC telemetry surely. Works OK with just Dshot selected in my case.

Yes, you're correct.. I thought the same.. But run a quick test before I wrote the original reply.
It was a case of just pressing SAVE after enabling a Dshot protocol, was not enough.. It requires SAVE and REBOOT for TURTLE to appear in the modes tab... And I presses SAVE and REBOOT after adding ESC telemetry on the Ports page.
